### ISSUE TITLE
can't use linqtoexcel

### ISSUE THREAD

**Issue by GoogleCodeExporter** [lholdenT/linqtoexcel#78]
```
What steps will reproduce the problem?
1.
 install linqtoexcel from nuget or manually

2.
 Dim book = New LinqToExcel.ExcelQueryFactory(archivoExcel.FileName)
                Dim PP = From X In book.Worksheet(Of solicitantesSiggan)()
3.

What is the expected output? What do you see instead?
it doesnt compile, book.Worksheet doesnt exists and it says "cant query an 
expression of type Object.....check for references for linq provider")
I got another projects(the sample provided in your site that works for example) 
that works but this version that comes from an import of an old 2.0 project 
doesnt works.

What version of the product are you using? On what operating system?

'LinqToExcel 1.6.5' - windows 7 - vs2010

Please provide any additional information below.

I got another projects(the sample provided in your site that works for example) 
that works but this project that comes from an import of an old 2.0 project 
doesnt works.
```

Original issue reported on code.google.com by `micuenta...@gmail.com` on 28 Nov 2012 at 3:24

**Comment by GoogleCodeExporter** [comment]
```
Can you zip up your project folder and attach the zipped file to this issue so 
I can take a look at it.
```

Original comment by `paulyo...@gmail.com` on 5 Dec 2012 at 8:37

**Comment by GoogleCodeExporter** [comment]
```
[deleted comment]
```

**Comment by GoogleCodeExporter** [comment]
```
problem solved, it doesnt works in 4.0, changed the project to use 3.5 
framework and it works.
```

Original comment by `micuenta...@gmail.com` on 7 Dec 2012 at 10:36

**Comment by GoogleCodeExporter** [comment]
Original comment by `paulyo...@gmail.com` on 7 Dec 2012 at 12:17
- Changed state: **Fixed**
